config.json

```diff
@@ -51,274 +51,5 @@
 			"balance": "1000000000000000000000"
 		}
 	],
-	"types": {
-		"HrmpChannelId": {
-			"sender": "u32",
-			"receiver": "u32"
-		},
-		"SignedAvailabilityBitfield": {
-			"payload": "BitVec",
-			"validator_index": "u32",
-			"signature": "Signature"
-		},
-		"SignedAvailabilityBitfields": "Vec<SignedAvailabilityBitfield>",
-		"ValidatorSignature": "Signature",
-		"HeadData": "Vec<u8>",
-		"CandidateDescriptor": {
-			"para_id": "u32",
-			"relay_parent": "Hash",
-			"collator": "Hash",
-			"persisted_validation_data_hash": "Hash",
-			"pov_hash": "Hash",
-			"erasure_root": "Hash",
-			"signature": "Signature"
-		},
-		"CandidateReceipt": {
-			"descriptor": "CandidateDescriptor",
-			"commitments_hash": "Hash"
-		},
-		"UpwardMessage": "Vec<u8>",
-		"OutboundHrmpMessage": {
-			"recipient": "u32",
-			"data": "Vec<u8>"
-		},
-		"ValidationCode": "Vec<u8>",
-		"CandidateCommitments": {
-			"upward_messages": "Vec<UpwardMessage>",
-			"horizontal_messages": "Vec<OutboundHrmpMessage>",
-			"erasure_root": "Hash",
-			"new_validation_code": "Option<ValidationCode>",
-			"head_data": "HeadData",
-			"processed_downward_messages": "u32",
-			"hrmp_watermark": "BlockNumber"
-		},
-		"CommittedCandidateReceipt": {
-			"descriptor": "CandidateDescriptor",
-			"commitments": "CandidateCommitments"
-		},
-		"ValidityAttestation": {
-			"_enum": {
-				"DummyOffsetBy1": "Raw",
-				"Implicit": "ValidatorSignature",
-				"Explicit": "ValidatorSignature"
-			}
-		},
-		"BackedCandidate": {
-			"candidate": "CommittedCandidateReceipt",
-			"validity_votes": "Vec<ValidityAttestation>",
-			"validator_indices": "BitVec"
-		},
-		"OriginKind": {
```

Comment:
I am not sure if we can remove all the below. That's the XCM stuff and I assume it is still not present on the polkadot.js side.

Reply:
Its all there: https://github.com/polkadot-js/api/blob/640ef4bdab2a15df8c2a752fe4ba75d37ed813bf/packages/types/src/interfaces/parachains/definitions.ts#L392 🙂

And is being actively maintained: https://github.com/polkadot-js/api/pull/3067

Reply:
fair enough then